src/charm.py

```diff
@@ -127,19 +150,48 @@ def _on_reset_instance_action(self, event: ActionEvent) -> None:
         if not container.can_connect():
             event.fail("Failed to connect to container")
             return
-        self.model.unit.status = ops.MaintenanceStatus("Resetting Synapse instance")
+        logger.info("Replan service to not restart")
```

Comment:
Perhaps it is worth explaining why the restart is not needed?

Reply:
Good point, I added a comment about it.